src/fill-zulipdb-with-msgs-deploy.mjs

```diff
@@ -0,0 +1,191 @@
+// Reference to TypeScript definitions for IntelliSense in VSCode
+/// <reference path="../rnode-grpc-gen/js/rnode-grpc-js.d.ts" />
+
+import grpcLib from '@grpc/grpc-js';
+import { rhoParToJson } from '@tgrospic/rnode-grpc-js';
+import util from 'util';
+
+// RNode with environment parameters
+import { rnodeService } from './rnode-env.mjs';
+
+// Load .env file
+import { config } from 'dotenv';
+config();
+
+import postgres from 'postgres';
+
+/**
+  * @param {object} arg
+  * @param {typeof path} arg.path
+  */
+async function main(env, {grpcLib}) {
```

Comment:
TypeScript parameters defined in comments doesn't correspond to function parameters.

VSCode can show errors with `// @ts-check` comment specified on top of the file. But it's ok to remove types if you don't have TS support in VSCode.